manifest/resolver.go

```diff
@@ -50,6 +50,23 @@ type Config struct {
 	Arch string
 }
 
+// Platform describes a system where a package can be installed
+type Platform struct {
+	// OS is the operating system of the platform
+	OS string
+	// Arch is the CPU architecture of the platform
+	Arch string
+}
+
+// HermitPlatforms are core platforms officially supported by Hermit.
+// For a package to be considered fully compliant, these platforms need to be supported
+var HermitPlatforms = []Platform{
+	{"linux", "x86_64"},
```

Comment:
Shouldn't this be `amd64` and `arm64`, that's what Hermit itself uses (from Go)?

Reply:
👍  I might extract these to constants